lib/action_view/component/base.rb

```diff
@@ -14,6 +13,13 @@ class Base < ActionView::Base
       class_attribute :content_areas, default: []
       self.content_areas = [] # default doesn't work until Rails 5.2
 
+      # `validation_module` should be set to a string if present
+      class_attribute :validation_module, default: "ActiveModel::Validations"
+      self.validation_module = "ActiveModel::Validations" # default doesn't work until Rails 5.2
+
+      # when set to `true` validation does not run
+      class_attribute :skip_validation
```

Comment:
I think we should deprecate the usage of `skip_validation: false`, meaning that validations will be gone on the next major version of `actionview-component`. Other than that, this PR is 💯

Reply:
I think we just eliminate it entirely. I don't think we need to introduce this concept at all.